### ISSUE TITLE
ram being stuck at a certain amount 

### ISSUE THREAD

**Issue by h4mzaforever** [FTBTeam/FTB-App#152]
if your ram is stuck at a low amount and cant change it even with the slider in the ftb app and you make sure you have 64 bit java you may need to check enviroment variables and if there is  _JAVA_OPTS or similar then delete it that may solve your problem

**Comment by jikuja** [comment]
Full list of environment variables available here: https://github.com/Slowpoke101/FTBLaunch/blob/509c89ef5f5ef73f6d5de59d79d84359044459e4/src/main/java/net/ftb/minecraft/MCLauncher.java#L223-L238

**Comment by jikuja** [comment]
Let me rephrase the problem with this ticket:

**The problem**
If user or any of user's other application has set _JAVA_OPTIONS, JAVA_TOOL_OPTIONS or JAVA_OPTIONS environment variables JVM of the MC process will pick command line arguments from those environment variables.

E.g. some mouse utilities set `-Xmx500M` or even `-Xmx50M` and then minecraft will fail with OOME. 

This is **hard to debug** because person helping user must know about those environment variable or MC process' stdout must be visible in the log.

**Solutions**
I see two possible for this

1. Ask user to clear those environment variables in OS level
    * Changes memory allocation of other java based tools => increases amount of the used memory => really bad advice => unhappy users
    * Horrible UX
2. Remove or empty those environment variables before starting MC process(before staring mojang launcher)
    * Environment variable changes are contained only for MC related processes and other applications are running as their developer has designed.
3. Do nothing
    * UX?

**Comment by Cloudhunter** [comment]
I do think it is more of a Mojang launcher bug than ours, but if we can work around it easily we will.

**Comment by jikuja** [comment]
Workaround is fairly easy with ProcessBuilder. See https://github.com/Slowpoke101/FTBLaunch/blob/master/src/main/java/net/ftb/minecraft/MCLauncher.java#L342https://github.com/Slowpoke101/FTBLaunch/blob/master/src/main/java/net/ftb/minecraft/MCLauncher.java#L342 for reference.

**Comment by Cloudhunter** [comment]
Fixed in dev
